docs/painless/painless-contexts/painless-context-examples.asciidoc

```diff
@@ -0,0 +1,80 @@
+[[painless-context-examples]]
+=== Context examples
+
+Use the commands below to setup an Elasticsearch cluster to execute the example
+scripts for each context where an example is provided. Each example must be
+executed in the order of the table in the
```

Comment:
I'd flip this around and say something like, "To run the examples, you first need to index the sample data into Elasticsearch:" 

...and then step through that process.

Reply:
Fixed.